comps/llms/text-generation/vllm/ray/dependency/build_docker_vllmray.sh

```diff
@@ -1,7 +1,10 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-cd ../../../../../../
+CURRENT_DIR=$(pwd)
+
+# Go to top level directory of this repo and build the image
+cd "$(git rev-parse --show-toplevel)"
```

Comment:
If they checkout the archive rather than the repo and no `.git` folder is present then this may not work.
But as long as CI's pass, we should be ok.

Reply:
Okay